generative_ai/batch_predict/gemini_batch_predict.py

```diff
@@ -45,9 +47,9 @@ def batch_predict_gemini_createjob(input_uri: str, output_uri: str) -> BatchPred
 
     # Submit a batch prediction job with Gemini model
     batch_prediction_job = BatchPredictionJob.submit(
-        source_model="gemini-1.5-flash-001",
+        source_model="gemini-1.5-flash-002",
```

Comment:
Have we confirmed that 002 works with batch prediction?

Reply:
Short answer. No.

I am testing it for all feature, right now. If required, I am on standby to roll back ASAP.

Reply:
It's confirmed that batch prediction works with 002 models.